src/github.js

```diff
@@ -112,6 +112,20 @@ async function assign_reviewers(reviewers) {
   });
 }
 
+// https://docs.github.com/en/rest/teams/members?apiVersion=2022-11-28#list-team-members
+async function get_team_members(team) {
+  const context = get_context();
+  const octokit = get_octokit();
+  const org = core.getInput('org') || context.payload.repository.owner.login;
```

Comment:
Could you elaborate why we'd like to have this logic? If I'm correct,

- `core.getInput('org')` allows us to pas an orgnisation
- `context.payload.repository.owner.login` returns a username, not org name

Also, if we'd like to add a new input parameter, we should update the README, too.

Reply:
Ok, so I misunderstood the use of `core` and can remove that bit - we shouldn't need a new input.
`context.payload.repository.owner.login` returns the organization if the repo owner is an organization. If it returns a username, then this action would error out but afaict there is no way to get _just_ the organization name for a repository or to tell the difference between an org or username and leave a useful error message. 
Do you know of a way?